config.js

```diff
@@ -40,7 +40,8 @@ module.exports = {
       instagram: '#',
       line: '',
       gitlab: '',
-      weibo: ''
+      weibo: '',
+      codepen: '#'
```

Comment:
```suggestion
      codepen: ''
```

Reply:
Thanks for pull request. The demo should have a maximum of 6 icons.

Reply:
Fair enough. It's done. Thanks for the review.